masterlist.yaml

```
@@ -5660,6 +5660,15 @@ plugins:
       - 'RevampedReleveled.esp'
       - 'Skyrim Revamped - Complete Enemy Overhaul.esp'
 
+  - name: 'mihailbeholder.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/34389' ]
+    msg:
+      - <<: *wildEdits
```

It only needs the `<<:` if you're making a change to the alias subs or condition.

```suggestion
    msg: [ *wildEdits ]
```